sandbox/mooncake.py

```diff
@@ -180,20 +191,20 @@ def forward(self, x):
             Mixture weights, means and standard deviations of each Gaussian
             component, respectively.
         """
-        x = x.unsqueeze(1)  # Sinewave only has 1 channel
-        x = self.conv1(x)
+        h1 = self.conv1(x)
 
-        residual = x
+        y = h1
+        residual = h1
         for i in range(self.num_blocks):
             residual = self.blocks[i](residual)
-            x += residual
+            y += residual
 
-        z = F.relu(x)
-        z = self.conv2(z)
+        z = F.relu(y)
+        h2 = self.conv2(z)
```

Comment:
we go from h1 -> y -> z -> h2 

:face_with_head_bandage:

Reply:
I thought you said outputs of convs were `h`s??